src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/model/ProfilerContents.java

```diff
@@ -45,4 +45,12 @@ public final native boolean getCreateProfile() /*-{
    public final native String getName() /*-{
       return this.name;
    }-*/;
+   
+   public final native String getHtmlLocalPath() /*-{
+      return this.htmlLocalPath;
+   }-*/;
+   
+   public final native boolean isUserSaved() /*-{
```

Comment:
I don't have a problem with shuffling around shortcuts (we've historically
done quite a bit of that, especially for less frequently used commands).

One final tweak: I'd eliminate the icon for that command so that it's
display is consistent with the rest of the items on the menu.


On Fri, Mar 4, 2016 at 5:16 PM, Javier Luraschi <notifications@github.com>
wrote:

> In
> src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/model/ProfilerContents.java
> <https://github.com/rstudio/rstudio/pull/619#discussion_r55097221>:
>
> > @@ -45,4 +45,12 @@ public final native boolean getCreateProfile() /*-{
> >     public final native String getName() /*-{
> >        return this.name;
> >     }-*/;
> > +
> > +   public final native String getHtmlLocalPath() /*-{
> > +      return this.htmlLocalPath;
> > +   }-*/;
> > +
> > +   public final native boolean isUserSaved() /*-{
>
> Sort of, fix was not complete, now it is. This new property tracks the
> saved state of the document to allow users to see the "save" command
> enabled only once while their doc has never ben saved. Once it's saved, it
> can only ve saved again with the "save as" command. This matches the
> behavior of other docs in the IDE.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rstudio/pull/619/files#r55097221>.
>